bears/r/FormatRBear.py

```diff
@@ -1,24 +1,40 @@
-from coalib.bearlib.abstractions.Lint import Lint
-from coalib.bears.LocalBear import LocalBear
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
 
 
-class FormatRBear(Lint, LocalBear):
-    executable = "Rscript"
-    arguments = ("-e 'library(formatR)' "
-                 "-e 'formatR::tidy_source(\"{filename}\")'")
-    prerequisite_command = ['Rscript', '-e', "'library(formatR)'"]
-    prerequisite_fail_msg = "Please install formatR for this bear to work."
-    diff_message = "Formatting can be improved."
-    gives_corrected = True
+@linter(executable='Rscript',
+        output_format='regex',
+        output_regex=r'(?P<line>\d+):'
+                     r'(?P<severity>\d+)'
+                     r'(?P<col>\d+)\s*(?P<message>.*)',
+        severity_map={"error": RESULT_SEVERITY.MAJOR,
+                      "warning": RESULT_SEVERITY.NORMAL,
+                      "warn": RESULT_SEVERITY.NORMAL,
+                      "information": RESULT_SEVERITY.INFO,
+                      "info": RESULT_SEVERITY.INFO})
+class FormatRBear:
     LANGUAGES = "R"
 
-    def process_output(self, output, filename, file):
-        output = output[:-1] + (output[-1].strip()+"\n",)
-        return Lint.process_output(self, output, filename, file)
-
-    def run(self, filename, file):
-        '''
-        This bear checks and corrects formatting of R Code using
-        known formatR utility.
-        '''
-        return self.lint(filename, file)
+    @staticmethod
+    def create_arguments(file, filename, config_file,
+                         comment: bool=True,
+                         blank: bool=True,
+                         arrow: bool=False,
+                         brace: bool=False,
+                         indent: int=4,
+                         output: bool=True):
+        """
+        :param comment:  use to determine whether comments are
+                         kept or not
+        :param blank:    use to determine whether blank lines
+                         are kept or not
+        :param brace:    whether to put the left brace
+        :param indent:   number of space for indentation
```

Comment:
can you use `tab_width` here? We have that name in other bears and we're trying to keep parameter names consistent across bears so one setting works for all of them.